storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc

```diff
@@ -92,6 +92,16 @@ If you have problems setting up static PVs, see xref:../../storage/container_sto
 
 include::modules/persistent-storage-csi-efs-security.adoc[leveloffset=+1]
 
+== AWS EFS storage CSI usage metrics
```

Comment:
Not sure if this is a rule, but I am an advocate of not stacking headings. Is there an intro sentence you could add here. Or not....